pkg/virt-launcher/virtwrap/network/network.go

```diff
@@ -41,6 +43,11 @@ var NetworkInterfaceFactory = getNetworkClass
 
 var podInterfaceName = podInterface
 
+type PodCacheInterface struct {
+	Iface *v1.Interface
```

Comment:
can we put the json markers here.

`json:"iface,omitempty"`
`json:"podIP,omitempty"`